docs/frontmatter.md

```diff
@@ -260,6 +260,9 @@ output:
     title-prefix: "[ai] "           # Optional: prefix for PR titles
     labels: [automation, ai-agent]  # Optional: labels to attach to PRs
     draft: true                     # Optional: create as draft PR (defaults to true)
+  labels:
+    allowed: [triage, bug, enhancement] # Mandatory: allowed labels for addition
```

Comment:
It feels like if this is not specified, then the agentic part should be able to leaver the chosen labels as data.